code/modules/cards/singlecard.dm

```diff
@@ -48,6 +48,9 @@
 		if(parent_deck.holodeck)
 			flags_1 |= HOLOGRAM_1
 			parent_deck.holodeck.spawned += src
+	if(mapload)
+		//maploaded card needs to be faceup anyways, and doing this will give it its name and appearance properly
+		Flip(TRUE)
```

Comment:
named parameter

Reply:
yoooo i'm just gonna name the define

Reply:
not sure thats better

Reply:
also also it should be flip() not Flip(), s not consistent with the only other named proc in the file `getMarkedColor`

Reply:
flip rename can happen in the future